### PR TITLE
feat: added `slackRoles` access methods to configuration and model

### PR DESCRIPTION
**Pull request by ssilare-adobe**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

File: packages/spacecat-shared-data-access/src/index.d.ts [hunk 1]
```diff
@@ -421,6 +421,12 @@ export interface Configuration {
    */
   getHandler: (type) => object;
 
+  /**
+   * Retrieves the slack configuration.
```
Comment (solaris007) [inline packages/spacecat-shared-data-access/src/index.d.ts:425]: doc doesn't match what the method does

File: packages/spacecat-shared-data-access/src/models/configuration.js [hunk 2]
```diff
@@ -24,6 +25,10 @@ const Configuration = (data = {}) => {
     state.handlers = state.handlers || {};
     state.handlers[type] = { ...handlerData };
   };
+  self.getSlackRolesByAuditType = (type) => {
```
Comment (solaris007) [inline packages/spacecat-shared-data-access/src/models/configuration.js:28]: slack roles are not related to "audit types" - did you mean something like "getSlackRoleMembersByRole"?

File: packages/spacecat-shared-data-access/src/index.d.ts (hunk 1 above)
Comment (ssilare-adobe) [reply packages/spacecat-shared-data-access/src/index.d.ts]: changed it to "slack roles configuration"

File: packages/spacecat-shared-data-access/src/models/configuration.js (hunk 2 above)
Comment (ssilare-adobe) [reply packages/spacecat-shared-data-access/src/models/configuration.js]: changed

**Pull request merged**